src/Install/InstallServiceProvider.php

```diff
@@ -54,6 +57,12 @@ function () {
             return $this->getRoutes();
         });
 
+        $this->app->singleton(
+            'Flarum\Install\Database\Drivers',
+            function () use ($drivers) {
+                return $drivers;
+        });
```

Comment:
This can be simplified with `$this->app->instance('Flarum\Install\Database\Drivers', $drivers)`, by the way. :)